src/ParametricUtils.jl

```diff
@@ -700,13 +781,15 @@ Initialize the parametric solver data from a different solution in `fromkey`.
 function initParametricFrom!(fg::AbstractDFG, fromkey::Symbol = :default; parkey::Symbol = :parametric, onepoint=false)
   if onepoint
     for v in getVariables(fg)
+      isInitialized(v, parkey) && continue
```

Comment:
don't think we should do that.  If you init a graph based on other solveKey then that should be absolute.  Not have cases that silently skip since I expect the user is trying to force a particular initialization.

Reply:
```suggestion
```